lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -291,14 +291,19 @@ class MailboxDashBoardController extends ReloadableController {
   }
 
   void _handleComposerCache() async {
-    _getEmailCacheOnWebInteractor.execute().fold(
-      (failure) {},
-      (success) {
-        if (success is GetComposerCacheSuccess) {
-          goToComposer(ComposerArguments.fromSessionStorageBrowser(success.composerCache));
-        }
-      },
-    );
+    if (accountId.value == null || sessionCurrent == null) return;
+
+    _getEmailCacheOnWebInteractor
+      .execute(accountId.value!, sessionCurrent!.username)
+      .fold(
+        (failure) {
+          _removeComposerCacheOnWeb();
+        },
+        (success) {
+          if (success is GetComposerCacheSuccess) {
+            goToComposer(ComposerArguments.fromSessionStorageBrowser(success.composerCache));
+          }
+        });
```

Comment:
we should use `consumeState` for synchronized

Reply:
Done